SpaDES4Dummies.Rmd

```diff
@@ -10,116 +10,142 @@ output: html_document
 
 ## decide where you're working
+mainDir <- 'C:/Users/tmichele/Dropbox/Project SpaDES/SpaDES4Dummies/' # SET YOUR MAIN DIRECTORY HERE
 setPaths(cachePath = "cache",
          inputPath = "inputs",
          modulePath = "m",
          outputPath = "outputs")
+
 getPaths() ## check that this is what you wanted
 
 ## Let's create a self-contained module that will simulate the species' abundance for any given period of time and frequency.
 if(!dir.exists(file.path(getPaths()$modulePath, "speciesAbundance"))){
   newModule(name = "speciesAbundance", path = getPaths()$modulePath)
 }
-
 ```
 
-You will notice that `newModule` has created a module folder inside */m* that contains both the module *.R* script template, as well as the documentation template (the *.Rmd* file). Although we will not be discussing the *.Rmd* file, please bear in mind that this is a **fundamental** part of creating a reproducible and transparent module. The documentation should contain not only the description of the module, but also some simple examples of what it can do.
+You will notice that `newModule` has created a module folder (`speciesAbundance`) inside */m* that contains both the module *.R* script template, as well as the documentation template (the *.Rmd* file). Although we will not be discussing the *.Rmd* file, please bear in mind that this is a **fundamental** part of creating a reproducible and transparent module - check out the [Guide to Reproducible Code in Ecology and Evolution](http://www.britishecologicalsociety.org/wp-content/uploads/2017/12/guide-to-reproducible-code.pdf) from the British Ecological Society). The documentation should contain not only the description of the module, but also some simple examples of what it can do.
 
-`newModule` also created the folder */data* where data necessary to the module should be put in, and the folder */tests* that may contain testing scripts. We won't be using neither of them in this example. 
+`newModule` also created the folder */data* where data necessary to the module should be put in, and the folder */tests* that may contain testing scripts. We won't be using either of them in this example. 
 
 **/!\\ Attention /!\\** : `newModule` should only be run once, or separately in the console, otherwise you may lose your module edits by re-running it again - this is why I wrapped it in an `if` statement (smart eh?).
 
+Now go ahead, open the *speciesAbundance.R* script and check it out a little bit.
+
+***
+
+#### SPECIES ABUNDANCE (`speciesAbundance`) Module
 
-Now go ahead and open the *speciesAbundance.R* script. I know, I know... It seems confusing and there's a lot of stuff in that template! 
+I know, I know... It seems confusing and there's a lot of stuff in that template! But I will describe it step by step. We will follow a different order from the `speciesAbundance.R` script file. We divide the Module script into 4 chunks: 
```

Comment:
Instead:
We'll go through it step by step (although not necessarily following the order of the script). The module script can be divided into 4 parts: